Lib/python/std_filesystem.i

```diff
@@ -0,0 +1,85 @@
+/*
+  filesystem::path
+*/
+%{
+#if __cplusplus >= 201703L
+#include <filesystem>
+#include <Python.h>
+#endif // __cplusplus >= 201703L
+%}
+
+%fragment("common", "header") {
```

Comment:
The fragment name here ("common") seems too generic.  We don't seem to really have a consistent naming convention for fragments containing helper functions, but something like `"SWIG_std_filesystem"` would at least be unlikely to collide with an unrelated fragment in a user interface file.